docs/DeveloperGuide.adoc

```diff
@@ -447,6 +447,55 @@ image::UndoRedoActivityDiagram.png[width="650"]
 ** Pros: We do not need to maintain a separate list, and just reuse what is already in the codebase.
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
+// tag::undoredo[]
+
+=== Import feature
+==== Current Implementation
+The import contacts feature is facilitated by the new `Command`, `import`. It adds a list of contacts from a properly formatted csv file
+to`AddressBook`. The rules pertaining to accepted formatting of csv files can be found in the user guide. Additionally, it
+implements the following operations:
+
+* `getFileFromUserInput(String)` -- gets a File from the path indicated by a user's text input.
+* `getFileFromFileBrowser()` -- gets a File via a file browser.
+* `parseFile(File) and parseLinesFromFile(BufferedReader)` -- parses the file from either of the above two methods. Prepares an
+arrayList of Persons to add to the contact list.
+
+Given below is an example usage scenario and how the import mechanism behaves at each step.
+
+Step 1. The user launches an application and there is either a list of existing contacts or the list is empty.
+
+Step 2. The user executes import command (i for shorthand). If the user ONLY types import, a file browser wlil pop up.
+If the user includes a file path, InsuRen will attempt to retrieve the file from the given path.
+
+image::Import.png[width="800"]
```

Comment:
This picture seems to be missing from the commit, either that or it's an extra line.